data/json/recipes/weapon/ranged.json

```diff
@@ -17,7 +17,68 @@
     "components": [
       [ ["nailgun", 1] ],
       [ ["pipe", 1] ],
-      [ ["2x4", 2], [ "stick", 3 ] ]
+      [
```

Comment:
This file isn't yet covered by the linter. You should try and keep all alternative components on one line though